src/ops/commit.rs

```diff
@@ -89,3 +89,43 @@ fn commit_fixup_cmd(args: &[OsString], rev: &OsStr) -> Command {
     cmd.args(args);
     cmd
 }
+
+#[derive(Display)]
+#[display(fmt = "Commit instant fixup")]
```

Comment:
I would propose we call it simply `Instant Fixup` and the other one simply `Fixup` since it's only popping on the `Commit` menu. WDYT?

Reply:
Yea I've been wanting to rename every action in every menu actually, but it's been always pushed forward. Let's merge this and I'll have a look at it before release :)